### PR TITLE
feat: Improve tx duration estimate

### PR DESCRIPTION
**Pull request by NunoAlexandre**
Transfers from DOT -> ETH take 30 min to 4 hours but currently we just '~30min' for both transfer directions which is confusing and misleading for users.

Here we show `~30 min to 4 hours` for `DOT -> ETH` and `~30 min` for `ETH -> DOT`.

### CODE REVIEW

File: app/src/components/FeesPreview.tsx [hunk 1]
```diff
@@ -41,7 +42,7 @@ const FeesPreview: FC<FeesPreviewProps> = ({ loading, fees, hidden }) => {
               )}
             </div>
             <div className="flex items-center">
-              <div className="text-turtle-foreground">~30 mins</div>
+              {estimate && <div className="text-turtle-foreground">{estimate}</div>}
```
Comment (noahjoeris) [inline app/src/components/FeesPreview.tsx:45]: Do you think we can name it `durationEstimate` to keep things clear

File: app/src/utils/transfer.ts [hunk 2]
```diff
@@ -131,3 +132,24 @@ export function getExplorerLink(transfer: StoredTransfer): string | undefined {
       console.log(`Unsupported network: ${network}`)
   }
 }
+
+//todo(team): query the right sdk to get the appropriate real-time estimate
+export function getEstimate(direction: Direction): string {
```
Comment (noahjoeris) [inline app/src/utils/transfer.ts:137]: and here maybe `getDurationEstimate`

**Review by noahjoeris — APPROVED** [review]
Nice. I left some minor requests. Feel free to ignore.

File: app/src/components/FeesPreview.tsx (hunk 1 above)
Comment (NunoAlexandre) [reply app/src/components/FeesPreview.tsx]: Agreed :) pushed 👍

**Pull request merged**